src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -35,6 +35,10 @@ export default class SecurityCache extends Ressource {
         );
     }
 
+    isListSecurityIdentitiesSupported() {
```

Comment:
```suggestion
    listSupportedSecurityIdentities() {
```

Reply:
The call returns if it's possible to listSecurityIdentities or not, it's an organization level boolean so either all identities of the org can be filtered or not, it could be renamed to `isListingSupportedSecurityIdentitiesSupported` if it makes things clearer

Reply:
oh I see 👍 I think it would indeed be clearer. Either that or a jsdocs comment over the method that adds more info to avoid people like me from thinking its for something else.

Reply:
More highlevel question: shouldn't that information be in the organization license thought?